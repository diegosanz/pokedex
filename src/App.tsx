import { FC } from "react";
import styles from "./styles.module.scss";

const App: FC = () => {
  console.log(styles);

  return <h1 className={styles.prueba}>Hello world</h1>;
};

export default App;
