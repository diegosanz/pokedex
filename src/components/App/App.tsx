import Router from '@components/Router/Router'
import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
import { FC } from 'react'

const queryClient = new QueryClient()

const App: FC = () => {
  return (
    <QueryClientProvider client={queryClient}>
      <Router />
    </QueryClientProvider>
  )
}

export default App
