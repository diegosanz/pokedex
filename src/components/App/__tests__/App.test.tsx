import { render } from '@testing-library/react'
import React from 'react'
import App from '../App'

describe('App', async () => {
  const setup = () => {
    return render(<App />)
  }

  it('Renders correctly and unmount correctly', () => {
    const { unmount } = setup()
    unmount()
  })

  it('Match Snapshot', () => {
    const { container } = setup()
    expect(container).toMatchSnapshot()
  })
})
