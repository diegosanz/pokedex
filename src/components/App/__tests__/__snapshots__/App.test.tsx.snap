// Vitest Snapshot v1

exports[`App > Match Snapshot 1`] = `
<div>
  <div
    class="_wrapper_a7a865"
  >
    <div
      class="_pokedexCase_a7a865"
    >
      <div
        class="_pokedexScreenFrame_a7a865"
      >
        <div
          class="_pokedexScreen_a7a865"
        >
          <div
            class="_screenHeader_41c9e3"
          >
            <a
              aria-current="page"
              class="_homeLinkHidden_41c9e3"
              href="/"
            >
              Go 
              <br />
               Home
            </a>
            <input
              class="_search_7818f8"
              placeholder="Search a Pokémon..."
              type="text"
            />
          </div>
          Loading...
        </div>
      </div>
      <div
        class="_brand_a7a865"
      >
        Diego Sanz Technologies ®
      </div>
    </div>
  </div>
</div>
`;
